internal/pkg/cli/task_run.go

```diff
@@ -481,6 +498,64 @@ func (o *runTaskOpts) Execute() error {
 	return nil
 }
 
+func (o *runTaskOpts) generateCommand() error {
+	command, err := o.runTaskCommand()
+	if err != nil {
+		return err
+	}
+	log.Infoln(command.String())
+	return nil
+}
+
+func (o *runTaskOpts) runTaskCommand() (*ecs.RunTaskRequest, error) {
+	var cmd *ecs.RunTaskRequest
+	sess, err := sessions.NewProvider().Default()
+	if err != nil {
+		return nil, fmt.Errorf("get default session: %s", err)
+	}
+
+	if arn.IsARN(o.generateCommandTarget) {
+		return o.runTaskCommandFromARN(sess)
+	}
+
+	parts := strings.Split(o.generateCommandTarget, "/")
+	switch len(parts) {
+	case 2:
+		clusterName, serviceName := parts[0], parts[1]
+		cmd, err = ecs.RunTaskRequestFromECSService(awsecs.New(sess), clusterName, serviceName)
+		if err != nil {
+			return nil, fmt.Errorf("generate task run command from ECS service: %w", err)
+		}
+	case 3:
+		appName, envName, serviceName := parts[0], parts[1], parts[2]
+		cmd, err = ecs.RunTaskRequestFromService(ecs.New(sess), appName, envName, serviceName)
+		if err != nil {
+			return nil, fmt.Errorf("generate task run command from service: %w", err)
+		}
+	default:
+		return nil, errors.New("invalid input to --generate-cmd: must be of the form <cluster>/<service> or <app>/<env>/<workload>")
```

Comment:
same as above: "workload" instead of "service" for the non-Copilot part of the error message?